.gitignore

```diff
@@ -9,38 +9,38 @@ venv
 *.log
 
 # User-specific stuff
-.idea/**/workspace.xml
```

Comment:
I'm note sure if newer versions of IDEA work differently (I'm still on 2020.1), but all the autogenerated files in .idea-directories of the module directories were annoying when moving between branches. I hope this change doesn't break anything.

Reply:
That's interesting, because i have only one .idea directory in the root directory, and i have various installations. Let's check this on your machine...

Reply:
I upgraded to IntelliJ 2021.3. If I revert to the original state of .gitignore, I see the following:
![image](https://user-images.githubusercontent.com/65666701/148558467-647311b9-8d30-4340-aaac-0e25a3ce4c43.png)